### PR TITLE
Add tourism=gallery

### LINKED ISSUE #990: add art galleries
* **What did you see?**
I did not see art galleries
* **What did you expect to see?**
Art galleries
* **What map location are you having problems with?** _City and country are helpful, as well as tile coordinates or latitude / longitude._  
Here are several from overpass:  [California Galleries](http://overpass-turbo.eu/map.html?Q=%2F*%0AThis%20query%20looks%20for%20nodes%2C%20ways%20and%20relations%20%0Awith%20the%20given%20key%2Fvalue%20combination.%0AChoose%20your%20region%20and%20hit%20the%20Run%20button%20above!%0A*%2F%0A%5Bout%3Ajson%5D%5Btimeout%3A25%5D%3B%0A%2F%2F%20gather%20results%0A(%0A%20%20%2F%2F%20query%20part%20for%3A%20“tourism%3Dgallery”%0A%20%20node%5B%22tourism%22%3D%22gallery%22%5D(31.05293398570514%2C-127.35351562499999%2C43.29320031385282%2C-111.24755859375)%3B%0A%20%20way%5B%22tourism%22%3D%22gallery%22%5D(31.05293398570514%2C-127.35351562499999%2C43.29320031385282%2C-111.24755859375)%3B%0A%20%20relation%5B%22tourism%22%3D%22gallery%22%5D(31.05293398570514%2C-127.35351562499999%2C43.29320031385282%2C-111.24755859375)%3B%0A)%3B%0A%2F%2F%20print%20results%0Aout%20body%3B%0A%3E%3B%0Aout%20skel%20qt%3B)
* **Screenshot?** 

One expected at Minna & 2nd: 
overpass example
missing in zinc example

Link to map above:  [Targeted Editing - Collections Map](https://mapzen-data.github.io/targeted-editing/te-collections/map/#19/37.78735/-122.39957)

* **OpenStreetMap Tags** - link to queries if helpful: [OpenStreetMap gallery queries](https://github.com/mapzen-data/targeted-editing/blob/gh-pages/queries/collections_sql.sql#L4)
  - Identified appropriate tag
    - [`tourism=gallery`](http://wiki.openstreetmap.org/wiki/Tag:tourism%3Dgallery)

  - Variants found with tag<b>info</b>
    - `amenity=gallery`
    - `amenity=art_gallery`
    - `amenity=arts_gallery`
    - `amenity=contemporary_art_gallery`
    - `building=gallery`
    - `gallery=*`
    - `tourism=art_gallery`
    - `tourism=Art Gallery`
    - `tourism=Art_gallery`
    - `museum=gallery`

### PR DESCRIPTION
**Pull request by iandees**
- [x] Update tests
- [x] Update data migrations
- [x] Update docs

Connects to #990.

### CODE REVIEW

**Comment by iandees** [comment]
This is ready for review, @nvkelso @rmarianski.

**Comment by iandees** [comment]
Woop! I didn't include the doc update. One moment.

**Comment by iandees** [comment]
Ok, ready to go.

File: data/migrations/v1.0.0-point.sql [hunk 1]
```diff
@@ -8,7 +8,8 @@ UPDATE
      tags -> 'amenity' = 'grave_yard' OR
      tags -> 'boundary' IN ('national_park', 'protected_area') OR
      tags -> 'power' IN ('plant', 'substation') OR
-     tags -> 'natural' IN ('wood', 'forest'))
+     tags -> 'natural' IN ('wood', 'forest') OR
+     tags -> 'tourism' = 'gallery')
```
Comment (rmarianski) [inline data/migrations/v1.0.0-point.sql]: Let's ensure that the polygon table also includes this `mz_poi_min_zoom` migration. Generally speaking, the `mz_poi_min_zoom` updates in the pois table should also be reflected in the polygon table. I forgot about this when I added this migration in the first place.

File: yaml/pois.yaml [hunk 2]
```diff
@@ -260,6 +260,9 @@ filters:
   - filter: {tourism: winery}
     min_zoom: GREATEST(14, LEAST(zoom + 2.85, 17))
     output: {kind: winery}
+  - filter: {tourism: gallery}
+    min_zoom: GREATEST(14, LEAST(zoom, 17))
```
Comment (rmarianski) [inline yaml/pois.yaml]: @nvkelso can you verify that it's ok for huge art galleries to start appearing at z14?

Comment (iandees) [inline yaml/pois.yaml]: He requested 15 in #990. I guess I was confused about this notation here. I'll update it to 15.

Comment (rmarianski) [inline yaml/pois.yaml]: Ah ok, sounds good. :+1:

File: data/migrations/v1.0.0-point.sql (hunk 1 above)
Comment (iandees) [inline data/migrations/v1.0.0-point.sql]: Done!

**Comment by rmarianski** [comment]
:+1:

File: integration-test/990-add-art-galleries.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# http://www.openstreetmap.org/node/2026996113
+assert_has_feature(
+    17, 20971, 50656, 'pois',
```
Comment (nvkelso) [inline integration-test/990-add-art-galleries.py]: Our max_zoom is 16, so you'd test the zoom 16 tile and see if the min_zoom on the feature is 17.

File: integration-test/990-add-art-galleries.py [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+    { 'id': 2026996113, 'kind': 'gallery' })
+
+# http://www.openstreetmap.org/way/31510288
+assert_has_feature(
+    17, 65485, 43583, 'pois',
```
Comment (nvkelso) [inline integration-test/990-add-art-galleries.py]: This one is an area and should be visible in a zoom 15 tile (not zoom 17).

File: yaml/pois.yaml [hunk 5]
```diff
@@ -260,6 +260,9 @@ filters:
   - filter: {tourism: winery}
     min_zoom: GREATEST(14, LEAST(zoom + 2.85, 17))
     output: {kind: winery}
+  - filter: {tourism: gallery}
+    min_zoom: GREATEST(15, LEAST(zoom, 17))
```
Comment (nvkelso) [inline yaml/pois.yaml]: I suspect this is more like museums, merged with your logic something like?

`min_zoom: GREATEST(15, LEAST(zoom + 1.43, 17))`

File: integration-test/990-add-art-galleries.py [hunk 6]
```diff
@@ -0,0 +1,9 @@
+# http://www.openstreetmap.org/node/2026996113
+assert_has_feature(
+    16, 10485, 25328, 'pois',
+    { 'id': 2026996113, 'kind': 'gallery', 'min_zoom': 16 })
```
Comment (nvkelso) [inline integration-test/990-add-art-galleries.py]: I suspect this should be `'min_zoom': 17`.

**Comment by nvkelso** [comment]
This is ready to merge.

**Pull request merged**